packages/server/src/plugin/landingPage/default/getEmbeddedHTML.ts

```diff
@@ -42,7 +42,7 @@ export const getEmbeddedExplorerHTML = (
 
     endpointUrl: string;
 
-    includeCookies?: boolean; // defaults to 'false'
+    hideCookieToggle?: boolean; // defaults to 'true'
   }
```

Comment:
@esilverm @trevor-scheer we don't support `hideCookieToggle` for the embedded Explorer, only Sandbox. 

This section of code should be reverted iirc. I am adding it to this PR https://github.com/apollographql/apollo-server/pull/7431